core/market/src/rest_api/common.rs

```diff
@@ -41,3 +45,28 @@ async fn get_agreement(
         Err(AgreementError::Internal(format!("We found ")))
     }
 }
+
+#[actix_web::get("/agreements/events")]
+async fn collect_agreement_events(
+    market: Data<Arc<MarketService>>,
+    query: Query<QueryAgreementEvents>,
+    id: Identity,
+) -> impl Responder {
+    let timeout: f32 = query.timeout;
+    let after_timestamp = query
+        .after_timestamp
+        .unwrap_or(Utc.ymd(1970, 1, 1).and_hms(0, 0, 0));
```

Comment:
use golem crowfunding date as `zero` time

Reply:
Fixed in #793